plugins/action_update_review_status/src/updateReviews.ts

```diff
@@ -17,7 +17,8 @@ async function updateReviews(input: any, DBConnect: any) {
 
   const { applicationId, applicationData, changedApplicationResponses = [] } = input
   const stageId = input?.stageId || applicationData.stageId
-  const level = input?.level || applicationData?.reviewData?.level || 1
+  const currentReviewLevel = applicationData?.reviewData?.levelNumber
+  const level = input?.level || currentReviewLevel ? currentReviewLevel + 1 : 1
```

Comment:
This will need to be adjusted slightly after #315 is merged, FYI (unless you merge this first)

Reply:
done, thanks for heads up